src/components/cc-pricing-page/cc-pricing-page.js

```diff
@@ -1,146 +1,189 @@
 import { css, html, LitElement } from 'lit';
-import '../cc-pricing-header/cc-pricing-header.js';
-import '../cc-pricing-product/cc-pricing-product.js';
-import '../cc-pricing-estimation/cc-pricing-estimation.js';
-import { dispatchCustomEvent } from '../../lib/events.js';
 
 /** @type {Currency} */
-const CURRENCY_EUR = { code: 'EUR', changeRate: 1 };
+const DEFAULT_CURRENCY = { code: 'EUR', changeRate: 1 };
+/** @type {Temporality} */
+const DEFAULT_TEMPORALITY = { type: '30-days', digits: 2 };
```

Comment:
question: I have already seen this in `cc-pricing-header`. Could we mutualise this constant?